docs/service-hub-context-builder-guide.md

```diff
@@ -0,0 +1,158 @@
+<!-- This doc is intent to replace the old management-sdk-guide -->
```

Comment:
directly change the management sdk guide, mark the SDK version for the old usage, and add a migration guide?